controls/sidedrawer/configuration.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Configuration
+page_title: .NET MAUI SideDrawer Documentation | Configuration
+description: Check our &quot;configuration&quot; documentation article for Telerik UI for .NET MAUI SideDrawer control.
+position: 2
+slug: sidedrawer-features-configuration
+---
+
+# Configuration
+
+This article will explain all configuration options that SideDrawer control provides.
+
+* `DrawerContent` (`View`)&mdash; Specifies the drawer (initially hidden) content.
```

Comment:
remove the space between the property and its type and the dash and explanation. Apply to app

how it should be: `DrawerContent`(`View`)&mdash;Specifies the drawer (initially hidden) content.